app/react-native/src/preview/components/OnDeviceUI/index.js

```diff
@@ -3,11 +3,11 @@ import PropTypes from 'prop-types';
 import { Keyboard, KeyboardAvoidingView, Platform, Animated, TouchableOpacity } from 'react-native';
 import Events from '@storybook/core-events';
 
-import StoryListView from '../StoryListView';
-import StoryView from '../StoryView';
-import Addons from './addons';
+import StoryListView from '../StoryListView/index';
```

Comment:
Isn't `index` implicit?

Reply:
Yes, but I don't recommend relying on that behavior

Reply:
Why not? Doesn't the majority of JS code (including most of the storybook codebase) rely on that?

Reply:
removed the `/index` since it is implicit and is consistent with the rest of the codebase